src/sentry/incidents/logic.py

```diff
@@ -652,7 +652,10 @@ def create_alert_rule(
                 )
 
             try:
```

Comment:
👌 

So we've covered `create_alert_rule`,
Lets also be sure to capture `update_alert_rule` as well

Reply:
I'll work in that in the next PR after Colleen's changes to `update_alert_rule` land